sdk/servicebus/service-bus/samples-dev/topicSubscriptionWithRuleOperationsSample.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT Licence.
+
+/**
+ * This sample demonstrates the topic filter functionality in Service Bus.
+ * It follows similar functionality to TopicSubscriptionWithRuleOperationsSample/program.cs
```

Comment:
Might be better to mention the link pointing to the sample or not mention at all.